src/components/ProgressIndicator/progressIndicator.spec.js

```diff
@@ -73,15 +75,37 @@ describe('The progress indicator component', () => {
     expect(stepPageItem).toBeVisible();
     const summaryPageItem = await screen.findByText('Summary');
     expect(summaryPageItem).toBeVisible();
-    const confirmationPageItem = await screen.findByText('Confirmation');
-    expect(confirmationPageItem).toBeVisible();
+  });
+
+  it('displays the available submission/form steps and hardcoded steps (with payment)', async () => {
+    mswServer.use(mockSubmissionPost(buildSubmission()));
+    const user = userEvent.setup({delay: null});
+    const form = getDefaultFactory({...FORM_DEFAULTS, paymentRequired: true})();
```

Comment:
could this not be done through `const form = buildForm({paymentRequired: true})`?